dev_guide/migrating_applications/S2I_tool.adoc

```diff
@@ -0,0 +1,64 @@
+[[dev-guide-s2i-tool]]
+= S2I Tool
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+xref:../../using_images/s2i_images/index.adoc#using-images-s2i-images-index][The
+Source-to-Image (S2I) tool] injects application source code into a Docker image
+and the final product is a new and ready-to-run Docker image that incorporates
+the builder image and built source code. The S2I tool can be installed in your
+local machine without {product-title} from
+link:https://github.com/openshift/source-to-image#installation[the repository].
+
+The S2I tool is a very powerful tool to test and verify your application and
+images locally before using them on {product-title}.
+
+[[s2i-creating-docker-image]]
+== Creating a Docker Image
+
+
+. Identify the builder image that is needed for the application. Red Hat offers
+multiple builder images for different languages including
+link:https://github.com/openshift?utf8=%E2%9C%93&query=s2i[Python, Ruby, Perl,
```

Comment:
The images were moved to `sclorg`, so `https://github.com/sclorg?query=s2i` might be a better link